### ISSUE TITLE
[Request] Auto-Disable dinput when using xinput

### ISSUE THREAD

**Issue by CaptainXLAB** [Ryochan7/DS4Windows#510]
I play some games like DmC Devil May Cry which get double input. 

With exclusive mode (and a cycle-controllers exe), it works but gives randomly missed vibrations. Rest of the time, controls on screen keep flashing between xinput (xbox controls) and dinput (generic 1,2,3 etc) and it's too annoying.

I couldn't find a way to disable dinput. I want xinput only. I tried disabling "wireless controller" in devices and printers but that disabled the controller completely. Any way to do this?

**Comment by Ryochan7** [comment]
DmC used to be a test game that I used. I had meant to play through the full campaign again using the DS4 but I never got around to it. As long as exclusive mode is working then DmC should only react to the virtual Xbox 360 controller. The double input problem would occur if the game is seeing the real DS4 device as well as the virtual Xbox 360 controller. Is the "Wireless Controller" device visible if you pull up the "Game controller settings" window? If it is then exclusive mode has failed and you have to disconnect and reconnect the DS4 to try again. There should also be a message in the DS4Windows log tab if attempting to obtain exclusive access failed. You might have to close some running programs if exclusive mode continues to fail.

**Comment by CaptainXLAB** [comment]
You're right. I usually always get an error in the log saying failed to start exclusive mode, but I just woke my comp up from sleep and this time there are no errors and it's working normally :smile: 

I'm on Windows 7 Ultimate SP1 x64 and I have no idea which program could cause exclusive mode to not work. I don't have steam/origin or anything running in background, I had discord but I closed that too. 

Any idea which program can be causing this?

DmC is my favorite game and I'm really good at it, been playing it every now and then since 2014 and I finished Hell and Hell without taking dying. And from all the controllers I've tried over the years (PS3/PS2/Xbox360/LogitechF710) I've gotta say, Dualshock 4 is the best so far. With a gyroscope and touchpad too, great grip on the controller, indented joysticks for better grip on them, RGB Lightbar and solid vibration

One strange thing - after I woke my comp up, the controller's battery went down to 81% and started charging again.
